lua/entities/gmod_wire_trigger.lua

```diff
@@ -0,0 +1,106 @@
+-- Wire Trigger created by mitterdoo
+AddCSLuaFile()
+DEFINE_BASECLASS( "base_wire_entity" )
+ENT.PrintName       = "Wire Trigger"
+ENT.RenderGroup		= RENDERGROUP_BOTH
+ENT.WireDebugName	= "Trigger"
+
+function ENT:SetupDataTables()
+	self:NetworkVar( "Vector", 0, "TriggerSize" )
+	self:NetworkVar( "Vector", 1, "TriggerOffset" )
+	self:NetworkVar( "Entity", 0, "TriggerEntity" )
+	self:NetworkVar( "Int", 0, "Filter" )
+	self:NetworkVar( "Bool", 0, "OwnerOnly" )
+end
+function ENT:Draw()
+	self:DrawModel()
+end
+
+if CLIENT then return end -- No more client
+
+function ENT:Initialize()
+	self:PhysicsInit( SOLID_VPHYSICS )
+	self:SetMoveType( MOVETYPE_VPHYSICS )
+	self:SetSolid( SOLID_VPHYSICS )
```

Comment:
These are already called in [`base_wire_entity.lua`](https://github.com/wiremod/wire/blob/master/lua/entities/base_wire_entity.lua#L347).